class BackException(Exception):
    pass


class ExitException(Exception):
    pass
